windows/win_feature.py

```diff
@@ -68,6 +68,15 @@
       - no
     default: null
     aliases: []
+  source:
+    description:
+      - Specify a source to install the feature from
+    required: false
+    choices: 
+      - {driveletter}:\sources\sxs
+      - \\{IP}\Share\sources\sxs
+    default: null
```

Comment:
When `required: false`, `default: null` is unnecessary (https://docs.ansible.com/ansible/developing_modules.html#module-checklist)